packages/web-components/fast-components/src/radio/radio.styles.ts

```diff
@@ -0,0 +1,137 @@
+import { css } from "@microsoft/fast-element";
+import { disabledCursor, display } from "../styles";
+import { focusVisible } from "../styles/focus";
+import { SystemColors } from "../styles/system-colors";
+import { heightNumber } from "../styles/size";
+import {
+    neutralFillInputHoverBehavior,
+    neutralFillInputRestBehavior,
+    neutralForegroundRestBehavior,
+    neutralOutlineHoverBehavior,
+    neutralOutlineRestBehavior,
+} from "../styles/recipes";
+
+export const RadioStyles = css`
+    ${display("inline-flex")} :host {
+        --input-size: calc((${heightNumber} * 0.5) + var(--design-unit));
+        align-items: center;
+        outline: none;
+        margin: calc(var(--design-unit) * 1px) 0;
+        ${
+            /*
+             * Chromium likes to select label text or the default slot when
+             * the radio button is clicked. Maybe there is a better solution here?
+             */ ""
+        } user-select: none;
+        position: "relative";
+        "flex-direction": "row";
+        transition: "all 0.2s ease-in-out";
+    }
+
+    .control {
+        position: relative;
+        width: calc(var(--input-size) * 1px);
+        height: calc(var(--input-size) * 1px);
+        box-sizing: border-box;
+        border-radius: 50%;
+        border: calc(var(--outline-width) * 1px) solid var(--neutral-outline-rest);
+        background: var(--neutral-fill-input-rest);
+        outline: none;
+        cursor: pointer;
+    }
+
+    .label {
+        font-family: var(--body-font);
+        color: var(--neutral-foreground-rest);
+        ${
+            /* Need to discuss with Brian how HorizontalSpacingNumber can work. https://github.com/microsoft/fast-dna/issues/2766 */ ""
+        } padding-inline-start: calc(var(--design-unit) * 2px + 2px);
+        margin-inline-end: calc(var(--design-unit) * 2px + 2px);
+        cursor: pointer;
+        ${
+            /* Font size is temporary - replace when adaptive typography is figured out */ ""
+        } font-size: calc(1rem + (var(--density) * 2px));
+    }
+
+    .checked-indicator {
```

Comment:
It seems that this `checked-indicator` is not centered.
![image](https://user-images.githubusercontent.com/37851214/80128847-1b94cf00-854b-11ea-80ae-0f080116cd82.png)

Reply:
prolly the errant " in the styles, looks centered now

Reply:
<img width="369" alt="Screen Shot 2020-04-23 at 10 31 07 AM" src="https://user-images.githubusercontent.com/25805778/80130687-dc1bb200-854d-11ea-9165-3cea42c8b684.png">